airbyte-commons-server/src/test/java/io/airbyte/commons/server/services/AirbyteGithubStoreTest.java

```diff
@@ -1,161 +0,0 @@
-/*
- * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.commons.server.services;
```

Comment:
Note: I removed these tests as the RemoteDefinitionProvider tests cover this class.

Let me know if I should still test this specific class further.